pkg/api/types.go

```diff
@@ -12,6 +12,30 @@ type ImageTag struct {
 	Timestamp    time.Time    `json:"timestamp"`
 	OS           OS           `json:"os,omitempty"`
 	Architecture Architecture `json:"architecture,omitempty"`
+
+	// If this is a Manifest list we need to keep them together
+	Children []*ImageTag `json:"children,omitempty"`
+}
```

Comment:
I suspect that it would be better to represent this as

```
type ImageTag struct {
    Tag string
    Manifests []Manifest
}

type Manifest {
   SHA string
   Timestamp time.Time
   OS OS
   Architecture Architecture
}
```